app/api/custom/tickets.py

```diff
@@ -0,0 +1,35 @@
+from flask import Blueprint
+from flask_jwt_extended import current_user, jwt_required
+from sqlalchemy.orm.exc import NoResultFound
+
+
+from app.api.auth import return_file
+from app.api.helpers.errors import ForbiddenError, NotFoundError
+from app.api.helpers.order import create_pdf_tickets_for_holder
+from app.api.helpers.storage import UPLOAD_PATHS
+from app.api.helpers.storage import generate_hash
+from app.models.order import Order
+
+ticket_blueprint = Blueprint('ticket_blueprint', __name__, url_prefix='/v1/tickets')
+
+
+@ticket_blueprint.route('/<string:order_identifier>')
+@jwt_required
+def ticket_attendee_authorized(order_identifier):
```

Comment:
This is also order API. Don't know why it is used as tickets API. The tickets in open event are not the same as tickets PDF of an order. 

We'll retain the endpoint for now, but add another endpoint, `/v1/orders/<identifier>/tickets`

Reply:
Please don't resolve reviews manually. They're resolved automatically when task is completed

Reply:
Still not moved

Reply:
@iamareebjamal I didn't delete the older endpoint because it would be in use on frontend and android.

Reply:
You have to move it to orders.py

Without deleting the endpoint. There's no need to duplicate it

Reply:
@iamareebjamal But what about the other endpoint `/v1/orders/<identifier>/tickets`?

Reply:
That will be added as well

Reply:
Endpoint != Function. So 2 endpoints doesn't mean 2 functions. 1 function can be mapped to many endpoints